Jellyfin.Api/Helpers/TranscodingJobHelper.cs

```diff
@@ -458,6 +458,7 @@ private void DeleteHlsPartialStreamFiles(string outputFilePath)
                     AudioChannels = state.OutputAudioChannels,
                     IsAudioDirect = EncodingHelper.IsCopyCodec(state.OutputAudioCodec),
                     IsVideoDirect = EncodingHelper.IsCopyCodec(state.OutputVideoCodec),
+                    IsHardwareEncode = !string.IsNullOrEmpty(_serverConfigurationManager.GetEncodingOptions().HardwareAccelerationType),
```

Comment:
```suggestion
                    HardwareAccelerationType = _serverConfigurationManager.GetEncodingOptions().HardwareAccelerationType,
```

Reply:
Done. Added a helper variable to make the line shorter and to handle cases where hardware acceleration is not used.